src/groupNormL2.jl

```diff
@@ -0,0 +1,88 @@
+# Group L2 norm (times a constant)
+
+export GroupNormL2
+
+"""
+**``L_2`` Group - norm**
+    GroupNormL2(λ = 1, g = 1, idx = [1, end])
+Returns the function
+```math
+f(x) =  \\sum\\_{i} \\lambda\\_{i}||x\\_{[i]}||\\_2)^{1/2}
+```
+for groups `x\\_{[i]}` and nonnegative weights `λ\\_i`.
+  Defaults to NormL2() in ProximalOperators if only 1 group is defined.
```

Comment:
I don't see that we fall back on ProximalOperator's NormL2 if only 1 group is defined.

Reply:
I suppose this text is a little misleading - if only 1 group is defined, then it is simply the l2-norm. I felt like it was difficult to actually default to the ProximalOperators version. Is there a way to do that? It is not obvious to me.

Reply:
Oh I see what you mean. How about something like "This operator reduces to the two norm if only one group is defined" or something like that?

Reply:
Pushed a new commit.